src/hyp3_isce2/__main__.py

```diff
@@ -46,10 +46,7 @@ def main():
             UserWarning,
         )
 
-    # NOTE: Cast to set because of: https://github.com/pypa/setuptools/issues/3649
-    # NOTE: Will need to update to `entry_points(group='hyp3', name=args.process)` when updating to python 3.10
-    eps = entry_points()['hyp3']
-    (process_entry_point,) = {process for process in eps if process.name == args.process}
+    process_entry_point = list(entry_points(group='hyp3', name=args.process))[0]
```

Comment:
Is the return not hashable anymore? I'd prefer to use the set for deduplication instead of just selecting the first one, so that you get an error if there are multiple, different entry points returned.